internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go

```diff
@@ -450,8 +465,9 @@ func resourceDataFactoryIntegrationRuntimeAzureSsisCreateUpdate(d *pluginsdk.Res
 		Description: &description,
 		Type:        datafactory.TypeBasicIntegrationRuntimeTypeManaged,
 		ManagedIntegrationRuntimeTypeProperties: &datafactory.ManagedIntegrationRuntimeTypeProperties{
-			ComputeProperties: expandDataFactoryIntegrationRuntimeAzureSsisComputeProperties(d),
-			SsisProperties:    expandDataFactoryIntegrationRuntimeAzureSsisProperties(d),
+			ComputeProperties:      expandDataFactoryIntegrationRuntimeAzureSsisComputeProperties(d),
+			SsisProperties:         expandDataFactoryIntegrationRuntimeAzureSsisProperties(d),
+			CustomerVirtualNetwork: expandDataFactoryIntegrationRuntimeCustomerVirtualNetwork(d.Get("express_vnet_integration").([]interface{})),
```

Comment:
I asked exactly that. It’s because that’s how the rest api implements it. https://docs.microsoft.com/en-us/rest/api/datafactory/integration-runtimes/create-or-update?tabs=HTTP#managedintegrationruntime

Reply:
Express aligns with what’s used in the azure portal ux. maybe that differentiation should be made out in the docs.

Reply:
Thats good enough for me! we generally try to match whats in the portal over the API. thanks